helperFunctions/SalesforceMessageHelpers.ts

```diff
@@ -7,20 +7,54 @@ import { ErrorLogs } from '../enum/ErrorLogs';
 import { getAppSettingValue } from '../lib/Settings';
 import { agentTypingListener, removeAgentTypingListener } from './AgentTypingHelper';
 import { sendLCMessage } from './LivechatMessageHelpers';
+import { retrievePersistentTokens } from './PersistenceHelpers';
 
-export async function messageFilter(app: IApp, modify: IModify, read: IRead, messageRoom: IRoom, LcAgent: IUser, messageArray: any) {
+export async function messageFilter(
```

Comment:
Can you please revert the formatting changes, this will create more changes than required when we will perform the catchup merge